docs/source/gbd2017_models/concept_models/vivarium_sanofi_zenon/concept_model.rst

```diff
@@ -123,12 +123,12 @@ The probability of side effects is also taken from the literature, and if a pers
 BAU parameter data tables
 +++++++++++++++++++++++++
 
-.. csv-table:: Current prescriptions
-   :file: current_rx.csv
-   :widths: 30, 20, 10, 10
+.. csv-table:: Table 1: Probability of having LDL-c measured
+   :file: prob_testing_ldlc.csv
+   :widths: 20, 10, 10
    :header-rows: 1
 
-.. csv-table:: Probability of prescription given high LDL-c
+.. csv-table:: Table 2: Probability of Rx given high LDL-c
    :file: prob_rx_given_high_ldlc.csv
```

Comment:
For background visits, if a patient is above the relevant threshold (4.9 mmol/L in BAU and according to the treatment algorithm involving SCORE, DM/CKD state, and SBP in the 2 intervention scenarios), they may or may not (therapeutic inertia) be given Rx.  Whether they are given Rx given that they are above the threshold is determined by the data in Table 2.

Reply:
Need input from Greg/Kelley/client - should we treat patients with 100% probability in the 2 intervention scenarios?  As is, we are not - we will use the prob(Rx|high LDL-C) from Table 2.